examples/node-agent-live/package.json

```diff
@@ -7,6 +7,7 @@
     "start": "node index.js"
   },
   "dependencies": {
+    "cross-fetch": "^4.1.0",
```

Comment:
yikes, big yikes. 

cross-fetch should be a devDependency so it doesn't get bundled with the code (it is only needed to run examples)

Reply:
fixed.